### ISSUE TITLE
ToDictionary overload ignores comparer

### ISSUE THREAD

**Issue by atifaziz** [atifaziz/LINQBridge#20]
```
ToDictionary overload taking keySelector and comparer as parameters totally 
ignores comparer.

```

Original issue reported on code.google.com by `mrdont@mail.ru` on 4 Apr 2011 at 8:13

**Comment by atifaziz** [comment]
Original comment by `azizatif` on 5 Apr 2011 at 12:11
- Changed state: **Accepted**

**Comment by atifaziz** [comment]
```
Please, incorporate this patch into repository.
```

Original comment by `mrdont@mail.ru` on 29 Apr 2011 at 10:53

Attachments:
- [GroupBy-Comparer.patch](https://storage.googleapis.com/google-code-attachments/linqbridge/issue-20/comment-2/GroupBy-Comparer.patch)

**Comment by atifaziz** [comment]
Original comment by `azizatif` on 30 Apr 2011 at 7:06
- Changed state: **Started**
- Added labels: **Milestone-Release1.3**

**Comment by atifaziz** [comment]
```
Fixed in c8b5b8ba1fbc[1]

[1] 
http://code.google.com/p/linqbridge/source/detail?r=c8b5b8ba1fbcfaeff85fdb73ff3b
41afac2effdf
```

Original comment by `azizatif` on 30 Apr 2011 at 7:07
- Changed state: **Fixed**
